locales/en-US/app.ftl

```diff
@@ -503,6 +503,7 @@ MenuButtons--metaOverheadStatistics-profiled-duration = Profiled Duration:
 ## These strings are used in the publishing panel.
 
 MenuButtons--publish--renderCheckbox-label-hidden-threads = Include hidden threads
+MenuButtons--publish--renderCheckbox-label-include-other-tabs = Include the data from other tabs
```

Comment:
It would probably work without the "the"? `Include data from other tabs`.

Reply:
Initially I did that, but below there's
```
MenuButtons--publish--renderCheckbox-label-private-browsing = Include the data from private browsing windows
```
that's why I kept it for consistency.
What do you think?

Reply:
All the other "Include xxx" don't have an article, I would fix the one about private browsing (no need for a new string ID).

Reply:
OK then, I'll do it for both in a separate patch so that I can update tests too.